protected.go

```diff
@@ -326,6 +326,60 @@ func (p *Protector) dialContext(op ops.Op, ctx context.Context, network, addr st
 	return conn.Conn, nil
 }
 
+func (p *Protector) ListenUDP(network string, laddr *net.UDPAddr) (*net.UDPConn, error) {
+	if laddr == nil {
+		laddr = &net.UDPAddr{}
+	}
+
+	op := ops.Begin("protected-listen-udp").Set("addr", laddr.String())
```

Comment:
This addr is a little different than the one in our dial ops because it's the listen rather than remote (dialed) address. Perhaps call it `laddr`?